Sources/Mantis/Protocols/CropViewProtocol.swift

```diff
@@ -48,6 +48,8 @@ protocol CropViewProtocol: UIView {
     
     func rotate(by angle: Angle)
     func makeTransformation() -> Transformation
+    func makeCropState() -> CropState
+    func makeCoreImageTransformInfo( ) -> (CGAffineTransform, CGRect)
```

Comment:
What is the purpose of this function?

Reply:
Removed from protocol and removed the function. It was for prototyping purposes.